routemaster/services/exception_loggers/new_relic.rb

```diff
@@ -0,0 +1,24 @@
+require 'singleton'
+require 'routemaster/services'
+require 'routemaster/mixins/log'
+
+module Routemaster
+  module Services
+    module ExceptionLoggers
+      class NewRelic
+        include Singleton
+        include Mixins::Log
+
+        def initialize
+          abort 'Please install and configure NewRelic first' unless ENV['NEW_RELIC_LICENSE_KEY']
+
+          require 'newrelic_rpm'
+        end
+
+        def process(e, env = ENV['RACK_ENV'])
```

Comment:
Indeed, but this is the [style used](https://github.com/deliveroo/routemaster/blob/master/routemaster/services/exception_loggers/honeybadger.rb#L24) [in all](https://github.com/deliveroo/routemaster/blob/master/routemaster/services/exception_loggers/print.rb#L12) [the others](https://github.com/deliveroo/routemaster/blob/master/routemaster/services/exception_loggers/sentry.rb#L28) so I figured I should keep the same here? Happy to swap them all to `_env` or similar in a new PR?